### PR TITLE
Add "Getting Read Access to Contracts" Guide

### PR DESCRIPTION
**Pull request by biancabuzea200**
| Q                        | A
| ------------------------ | ---
| Service                  | docs
| Fixed Issues?            | -
| Patch: Bug Fix?          | -
| Major: Breaking Change?  | -
| Minor: New Feature?      | -
| Tests Added + Pass?      | Yes
| Documentation PR         | `[skip-ci]`
| Any Dependency Changes?  | -

### Add "Getting Read Access to Contracts" under Guides section

### CODE REVIEW

**Review by WesleyCharlesBlake — APPROVED** [review]

File: docs/Developers/Guides/whitelistAddress.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+$ cast send 0x0dcc19657007713483a5ca76e6a7bbe5f56ea37d "selfKiss(address)()" <oracle address you want you use> --rpc-url $rpc
+```
+
+Where `0x0dcc19657007713483a5ca76e6a7bbe5f56ea37d` is the SelfKisser contract address.
+
+If you are looking for an Oracle address to quickly test out, feel free to use: Chronicle_ETH_USD_3	`0xdd6D76262Fd7BdDe428dcfCd94386EbAe0151603`.
+
+You can then verify your whitelist status by running:
+```bash
+$ cast call <oracle address> "tolled(address)" <your address> --rpc-url $rpc
```
Comment (jar-o) [inline docs/Developers/Guides/whitelistAddress.md:30]: if you revise this to return a `bool` time the output becomes `true|false` e.g.
```
cast call <oracle address> "tolled(address)(bool)" <your address> --rpc-url $rpc
true
```

File: docs/Developers/Guides/whitelistAddress.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+### Reading from Smart Contracts using the CLI 
+Once you are whitelisted, interacting with the oracle is easy. The `read()` function returns a uint value representing the data you are trying to get:
+
+```bash
+$ cast call 0xc8A1F9461115EF3C1E84Da6515A88Ea49CA97660 "read()(uint)" --rpc-url $rpc
+```
+
+Response:
+`1656870000000000045440`
```
Comment (jar-o) [inline docs/Developers/Guides/whitelistAddress.md:44]: we have other "read" functions too, but maybe not necessary for the scope of this doc? i.e. `tryRead`, `tryReadWithAge` etc

**Review by jar-o — DISMISSED** [review]
lgtm, left a couple of comments

File: docs/Developers/Guides/whitelistAddress.md (hunk 2 above)
Comment (biancabuzea200) [reply docs/Developers/Guides/whitelistAddress.md:43]: Thank you for bringing this up. I will leave it out of the scope for this one, but definitely integrate into a future one.

**Review by jar-o — APPROVED** [review]
lgtm

**Pull request merged**